hub/api/tests/test_api_with_compression.py

```diff
@@ -186,3 +186,15 @@ def test_chunkwise_compression(ds: Dataset, cat_path, flower_path):
     labels.extend(data)
     for row, label in zip(data, labels):
         np.testing.assert_array_equal(row, label.numpy())
+
+
+@enabled_datasets
+@pytest.mark.parametrize("compression", hub.compression.AUDIO_COMPRESSIONS)
+def test_audio(ds: Dataset, compression, audio_paths):
+    path = audio_paths[compression]
+    arr = None
+    ds.create_tensor("audio", htype="audio", sample_compression=compression)
+    with ds:
+        for _ in range(2):
+            ds.audio.append(hub.read(path))  # type: ignore
+    np.testing.assert_array_equal(ds.audio[0].numpy(), ds.audio[1].numpy())  # type: ignore
```

Comment:
hmm, i can see this going very wrong very fast

Reply:
could you have a better ground truth baseline? maybe even add an audio reader package for the requirements/tests.txt?